### PR TITLE
Added info about two flags (quiet/dry-run)

### PR DESCRIPTION
**Pull request by roykrikke**
Readme updated on request. I have also taken the liberty of adding dry-run flag description.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -70,6 +70,9 @@ Example:
 phockup ~/Pictures/camera ~/Pictures/sorted
 ```
 
+### Version
+If you want view the version of phockup use the flag `-v | --version`.
```
Comment (ivandokov) [inline readme.md:74]: If you want **to** view...

**Review by ivandokov — CHANGES_REQUESTED** [review]

File: readme.md (hunk 1 above)
Comment (roykrikke) [reply readme.md]: Done

**Comment by roykrikke** [comment]
I noticed that the latest update did not automatically come with "snap". Is that correct behavior?

**Comment by ivandokov** [comment]
Yes it is correct as new versions are deployed only by new tags.

**Pull request merged**